src/test/java/com/kodilla/ecommercee/domain/OrderEntityTestSuit.java

```diff
@@ -0,0 +1,116 @@
+package com.kodilla.ecommercee.domain;
+
+import com.kodilla.ecommercee.exception.OrderNotFoundException;
+import com.kodilla.ecommercee.service.CartService;
+import com.kodilla.ecommercee.service.OrderService;
+import com.kodilla.ecommercee.service.UserService;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import javax.transaction.Transactional;
+
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class OrderEntityTestSuit {
+    @Autowired
+    OrderService orderService;
+    @Autowired
+    UserService userService;
+    @Autowired
+    CartService cartService;
+
+    @Transactional
+    @Test
+    public void testFindAll() {
+        //Given
+        int ordersTableSizeBeforeTest = orderService.getOrderList().size();
+
+        //When
+        orderService.addOrder(new Order());
+        orderService.addOrder(new Order());
+        orderService.addOrder(new Order());
+
+        //Then
+        assertEquals(3, orderService.getOrderList().size() - ordersTableSizeBeforeTest);
```

Comment:
No to te testy nie są zgodne z tym co ostatnio sprawdzałem. Co prawda testowałem to i w obecnej konfiguracji H2 zwija transakcje po przeprowadzeniu testów, jednakże jeśli coś będzie w bazie danych to test ten może nie przejść, bo np. zwróci o jeden wynik więcej.

Zerknij tutaj:
https://github.com/pplutap/JDP-1901-01/pull/18/files#r256634167

Niby H2 startuje od 0 i tworzy strukturę itp... ale chłopaki poszli chyba w inną stronę zanim jeszcze ja zrobiłem review i dostałem info, iż tak ustaliliście. Co teraz?

Reply:
jak widać w teście pobieram ilość zamówień w bazie i odejmuje od tego ilość zamówień które znajdowały się bazie przed dodaniem zamówień więc problem zwracania innego wyniku jeśli baza zawiera jakieś wpisy jest wyeliminowany.